plugins/woocommerce/src/Internal/ReceiptRendering/Templates/order-receipt.php

```diff
@@ -0,0 +1,97 @@
+<html>
+<head>
+	<style>
```

Comment:
```suggestion
	<meta http-equiv="Content-Type" content="<?php bloginfo( 'html_type' ); ?>; charset=<?php echo get_option( 'blog_charset' ); ?>" />
	<style>
```

I ran into some character encoding issues at least once while testing receipt generation. It wasn't consistent, weirdly, but adding the above meta tag to the receipt file fixed it when it happened.